.tekton/push.yaml

```diff
@@ -22,7 +22,6 @@ spec:
     - name: infra-deployment-update-script
       value: |
         sed -i -e 's|\(https://github.com/redhat-appstudio/integration-service/.*?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/hacbs/integration-service/base/kustomization.yaml
```

Comment:
This is wrong. There is no `components/hacbs` dir in infra-deployments anymore

Reply:
Thanks! Updated.